gamemodes/terrortown/entities/weapons/weapon_zm_improvised.lua

```diff
@@ -264,7 +264,7 @@ function SWEP:SecondaryAttack()
         tr.Hit
         and IsValid(ply)
         and ply:IsPlayer()
-        and (owner:EyePos() - tr.HitPos):Length() < 100
+        and (owner:EyePos() - tr.HitPos):LengthSqr() < 10000 -- 100hu
```

Comment:
Is there a specific reason why working with the squared length is better here?

Reply:
LengthSqr is more efficient than Length because [Length uses the square root function which is an expensive process](https://wiki.facepunch.com/gmod/Vector:Length). Generally if you only need to calcuate if two vectors within range of each other (ie. you don't need a human readable distance in hammer units), LengthSqr should be used instead

Reply:
Good to know, never thought about this. We should probably adress this in our whole code base